Bottom: 70e5aa92b0f7b7bd5e46962b4d89379d36daa725
Top:    bc56257f04aa66cbdcbb09b964f3ad3a8fbbcef9
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 15:59:20 +0800

Refresh of autofs-5.0.7-fix-portmap-lookup.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index d6c4b5a..5164e70 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -280,7 +280,7 @@ static int rpc_do_create_client(struct sockaddr *addr, struct conn_info *info, i
 		in4_raddr->sin_port = htons(info->port);
 		slen = sizeof(struct sockaddr_in);
 		/* Use rpcbind v2 for AF_INET */
-		if (info->program == rpcb_prog)
+		if (info->program != PMAPVERS)
 			info->version = PMAPVERS;
 	} else if (addr->sa_family == AF_INET6) {
 		struct sockaddr_in6 *in6_raddr = (struct sockaddr_in6 *) addr;
